js/main.js

```diff
@@ -0,0 +1,119 @@
+        var user_id = "138698049@N03";
+        var format = '&format=json';
+        var url = "https://api.flickr.com/services/rest"+"?method="+method+"&api_key="+api_key+"&user_id="+user_id+format+'&nojsoncallback=1';
+
+        var xmlhttp = new XMLHttpRequest();
+
+        xmlhttp.onreadystatechange = function () {
+            var farm, server, id, title, secret, picUrl, list, img, items;
+
+            if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
+                //make an empty array
+                var pics = [];
+
+                //parse request to json
+                var myResp = JSON.parse(xmlhttp.response);
+                //console.log(myResp);
+
+                for(var i=0; i< myResp.photos.photo.length; i++){
+                    farm = myResp.photos.photo[i].farm;
+                    server = myResp.photos.photo[i].server;
+                    id = myResp.photos.photo[i].id;
+                    title = myResp.photos.photo[i].title;
+                    secret = myResp.photos.photo[i].secret;
+                    picUrl = "https://farm"+farm+".staticflickr.com/"+server+"/"+id+"_"+secret+"_z.jpg";
+
+                    //add each picture url to the pics array
+                    pics.push(picUrl);
+                }
+
+                for(var i=0; i<pics.length; i++){
+                    list = document.getElementById('list');
+                    img = '';
+                    for(var i = 0; i<pics.length; i++){
+                        list.innerHTML = list.innerHTML + '<li class='+'active'+ ' id=img'+[i]+'><img src=' + pics[i] + ' /></li>';
+                    }
+                }
+                items = document.getElementById('img0');
+                items.classList.remove('active');
+                items.classList.add('current');
+
+                getCarousel();
+            }
+        };
+        xmlhttp.open("Get", url, true);
+        xmlhttp.send();
+    }
+
+    function getCarousel(){
+        var box = document.getElementsByClassName('image-slider-wrapper');
+        var next = document.getElementById('next');
+        var counter = 0;
+        var prev = document.getElementById('prev');
+        var items = document.getElementsByTagName('img');
+        var amount = items.length;
+        var current = document.getElementsByClassName('current');
+
+        function navigate(direction){
+            if(direction === 1){
+                items = document.getElementById('img'+counter);
+                current = document.getElementsByClassName('current');
+                items.classList.remove('current');
+                items.classList.add('active');
+            }
+
+        }
+        next.addEventListener("click", function(event){
+            items = document.getElementById('img'+counter);
+            counter = counter+1;
+
+            if(counter <= amount-1){
+                items.classList.remove('current');
+                items.classList.add('active');
```

Comment:
My vanilla JS is a bit rusty, but are this chainable?

`items.classList.remove('current').add('active');`?